### PR TITLE
makes bandits loadouts matter more and buffs bog guard to last longer in fights, gives them meds too

### PR DESCRIPTION
**Pull request by bobjension**
no more crossbow spam yippie!!!, also increases their slots cuz why not?

### CODE REVIEW

File: code/game/gamemodes/roguetown/roguetown.dm [hunk 1]
```diff
@@ -197,7 +197,7 @@ var/global/list/roguegamemodes = list("Rebellion", "Vampire Lord", "Extended", "
 	"Knight")
 	var/num_bandits = 0
 	if(num_players() >= 10)
-		num_bandits = CLAMP(round(num_players() / 2), 15, 20)
+		num_bandits = CLAMP(round(num_players() / 2), 20, 25)
```
Comment (ChungusGamer666) [inline code/game/gamemodes/roguetown/roguetown.dm:200]: PLEASE. NO MORE. NO....

Comment (SELFHELL) [reply code/game/gamemodes/roguetown/roguetown.dm:200]: We can go higher.

**Comment by YosemiteYam** [comment]
Cloth sucks ass, everyone just rips two cloth from their shirts at roundstart anyway, give them a needle instead if you want to give "meds" though even that only takes a 20 seconds to get from the bushes to the south. 

If you want them to survive more, some health potions would actually help them since they don't have a cleric to save their ass during a bandit zerg rush at roundstart.

**Comment by Osmos123** [comment]
Fixes #369

**Comment by Gyrandola** [comment]
the bog guard - bandit powercreep will make us end up with 30 bandits and 25 bog guards all legendary swordsmen

**Comment by zakanater19** [comment]
in my mind i think bog guard SHOULD get overwhelmed and die, it's like their whole gimmick, after they die then the actual town guards and castle guards have to step up

**Comment by zakanater19** [comment]
based RNG

**Comment by CVSthePharmacy** [comment]
why the fuck are you nerfing bow and cross skill on them, that's what they're supposed to be good at

**Pull request closed without merging**